test/sql/columns.sql

```diff
@@ -21,3 +21,12 @@ INSERT INTO t (b) VALUES (1), (2);
 INSERT INTO t (b) SELECT 3;
 SELECT * FROM t;
 DROP TABLE t;
+
+-- Test RETURNING clause for INSERT.
+CREATE TABLE t (a int) USING columnstore;
+INSERT INTO t VALUES (123) RETURNING a + 1;
+DROP TABLE t;
+
+CREATE TABLE t (a int, b int DEFAULT 3) USING columnstore;
+INSERT INTO t VALUES (1, DEFAULT), (2, DEFAULT) RETURNING *;
+DROP TABLE t;
```

Comment:
`test/sql/columns.sql` is to test special columns, e.g. default value, identity column, generated column
`RETURNING` clause should have its own test